test/multiplier/multiplier.go

```diff
@@ -0,0 +1,135 @@
+package multiplier
+
+import (
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/sourcenetwork/corekv/test/action"
+)
+
+var activeMultipliers []Multiplier
+var availableMultipliers []Multiplier
+
+type Name = string
```

Comment:
todo: Please document why this alias is needed or useful.

Reply:
`Name` is a little more descriptive than `string`, especially for return params.

- [ ] Document `Name`